### DISCUSSION TITLE
A solution to run AsusOSD at startup with MyAsus uninstalled

### DISCUSSION THREAD

**Discussion by joolowweng** [seerge/g-helper#168] [Show and tell]
First I would like to appreciate author for developing this app.
For many of you who want to fully remove MyAsus App and still be able to have OSD functioning, this is something I figured out:
1. Go to 'C:\Windows\System32\DriverStore\FileRepository\asussci2.inf_amd64_bf969a2fcea35523\ASUSOptimization' if you have the latest Asus System Control Interface installed. 
2. Right click on AsusOSD and click on Properties. Go to the Compatibility tab, check the "Run this program as an administrator" option, and then click on Apply.
3. Run 'Task Scheduler' and create a new basic task, and you will be able to trigger AsusOSD.exe at log on.

**Comment by den7007** [comment]
Thanks for this! Just a small note for others - make sure to have "Run with highest privileges checked" - I was wondering why mine wouldn't work at first haha

**Reply by Metalogic999** [reply]
BTW should I also uncheck both "Start the task only if the computer is on AC Power" and "Stop of the computer switches to battery power"? As I guess you want the OSD regardless, unless it consumes a lot of resources?

**Comment by Metalogic999** [comment]
@joolow @den7007 Great, thanks!

**Reply by Metalogic999** [reply]
@joolow @den7007 BTW how can I check if this solution is working or not? I've created the Basic task in Task Scheduler and have both checked the "Run this program as administrator" in the Compatibility tab of AsusOSD.exe and also checked "Run with highest privileges checked" in the Basic task properties, but I don't think I'm seeing the OSD pop-ups displayed.
